ee/hogai/graph/insights/nodes.py

```diff
@@ -196,7 +267,7 @@ def _search_insights_iteratively(self, search_query: str) -> list[int]:
                             page_number = tool_call["args"]["page_number"]
 
                             if page_number == 0:
-                                tool_content = "Page 1 data was already provided in the initial context above."
+                                tool_content = "Page 0 data is already provided in the initial context above."
```

Comment:
Could you explain the numbering logic, please? I see `Page 0`, but `page_number+1`, so not sure what number the numeration starts with.